source/Octopus.Tentacle/Kubernetes/IKubernetesPodLogService.cs

```diff
@@ -61,6 +69,11 @@ async Task<(IReadOnlyCollection<ProcessOutput> Outputs, long NextSequenceNumber,
                     return await PodLogReader.ReadPodLogs(lastLogSequence, reader);
                 }
             }
+            
+            IReadOnlyCollection<ProcessOutput> ReadTentacleLogs()
```

Comment:
Not sure this method is serving a purpose

Reply:
It used to...until it got that small 😬